src/UniversalApp.js

```diff
@@ -39,6 +39,7 @@ const initServer = function(app, routes, apolloServerOptions, production) {
             req,
             typeDefs,
             resolvers,
+            cache: apolloServer.requestOptions.cache,
```

Comment:
Can we access `cache` option via `apolloServerOptions.cache`?

Reply:
Looks like apolloServer has some logic about on the default cache strategy.

For example, when we pass null to apolloServerOptions.cache, `InMemoryLRUCache` will be used as the default cache.

Either we need to mimic and replicate this logic in `serverRender` , or use the one directly from `apolloServer.requestOptions.cache`

Reference: https://github.com/apollographql/apollo-server/blob/dab850625a190c5602279769630631ae9a6e4fa4/packages/apollo-server-core/src/ApolloServer.ts#L209-L211

Reply:
You're right. We should use `apolloServer.requestOptions.cache`.